metadata.json

```diff
@@ -27,47 +27,42 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5",
```

Comment:
RH still supports RHEL 5 so I'd keep this in unless you actually drop compatibility code

Reply:
ah right. this is actually RHEL and not CentOS